src/domain/collaboration/callout-framing/callout.framing.service.ts

```diff
@@ -108,18 +108,12 @@ export class CalloutFramingService {
       );
     }
 
-    if (calloutFraming.whiteboard && calloutFramingData.whiteboard) {
-      calloutFraming.whiteboard = await this.whiteboardService.updateWhiteboard(
```

Comment:
how are you updating everything else besides the content in `framing.whiteboard` ?

Reply:
we aren't. but it's the whiteboard inside the template, we are not using that profile at all. It has just fixed data like displayName: 'Whiteboard Template', and it's not shown anywhere and it was not updated earlier either (the client wasn't sending `whiteboard.profile`. 
If we still want to update it, we can call UpdateWhiteboard mutation with the id of the whiteboard.